CHANGELOG.md

```diff
@@ -17,8 +17,8 @@ These should not affect the functionality of the module.
 
 [Full Changelog](https://github.com/voxpupuli/puppet-extlib/compare/v2.3.1...v3.0.0)
 
-In this release, all functions have been moved to the `extlib::` namespace and converted to the 4.X function API.  Non name-spaced versions are still available, but deprecated and will be removed in a future major release.

```

Comment:
I don't think this was 'trailing whitespace', as the linter said, but windows line endings in an otherwise unix line ending terminated file.
I've opened https://github.com/github-changelog-generator/github-changelog-generator/issues/715